src/osd/modules/render/drawbgfx.cpp

```diff
@@ -395,6 +395,17 @@ bool video_bgfx::set_platform_data(bgfx::PlatformData &platform_data, osd_window
 	platform_data.ndt = nullptr;
 	platform_data.nwh = GetOSWindow(dynamic_cast<mac_window_info const &>(window).platform_window());
 #else // defined(OSD_*)
+
+#if defined(SDLMAME_EMSCRIPTEN)
```

Comment:
Please go for #elif there, compilers can get cranky when you have a pile of stuff after a return and it's bad form in any case.

Reply:
Thanks for the suggestion. The emscripten specific code has been updated to an #elif statement, making the code align with the approach used by other system for setting up bgfx 👍